torch/quantization/QConfig.py

```diff
@@ -9,6 +9,9 @@
 default_qconfig = QConfig(default_weight_observer(),
                           default_observer())
 
+default_debug_qconfig = QConfig(default_weight_observer(),
```

Comment:
also I'm changing the order of activation and weight in qconfig, please use keyword arguments instead: https://github.com/pytorch/pytorch/pull/25950